src/layer/water.js

```diff
@@ -64,13 +64,66 @@ export const water = {
       0.3,
       1,
     ],
+    "fill-outline-color": Color.waterFillTranslucent,
```

Comment:
Because the waterline is under the water fill layer, only the half of the line that lies outside the waterbody would normally be visible. This outline doesn’t apply the waterline color. By applying some extra blue, it seems to help the waterline overcome the fill’s antialiasing and show through more prominently than it would on its own. The waterline relies on color instead of width because any additional width makes some waterways look cartoonish.

Reply:
Try experimenting on waterLine with (say) `{"line-width":1.25,"line-blur":0.25}`, especially at 1x vs 2x. Could smooth out jaggies and make it more subtle, or just look terrible.

Reply:
The `line-width` makes the line darker (not wider), but the addition of a `line-blur` seems to cancel out the change on a 2× display. I don’t see any difference from the original state of this branch. Is there a particular area I should be testing?

Reply:
Not sure, just an idea. The lakes in eastern Ohio are pretty noisy though.

Reply:
Sorry, I’m not seeing the `line-width` and `line-blur` make any difference there either. If you mean that the waterlines make the reservoirs in eastern Ohio look much more prominent, I’m seeing the same thing in North Texas, especially when introducing waterlines below z8: https://github.com/ZeLonewolf/openstreetmap-americana/pull/610#discussion_r1045143790.